src/osd/ReplicatedBackend.cc

```diff
@@ -261,7 +261,7 @@ int ReplicatedBackend::objects_readv_sync(
   uint32_t op_flags,
   bufferlist *bl)
 {
-  interval_set<uint64_t> im(m);
+  interval_set<uint64_t> im(std::move(m));
```

Comment:
i'd suggest change the signature of `ReplicatedBackend::objects_readv_sync()` like
```c++
/// ...
  map<uint64_t, uint64_t>&& m,
```
so the caller will have to pass by rvalue reference, and the caller will be aware that `m` will be moved away after the call.